src/cio.ts

```diff
@@ -9,12 +9,25 @@ import {
   getShortGenericInviteLink,
 } from './common';
 import { FastifyBaseLogger } from 'fastify';
+import { UserCompany } from './entity/UserCompany';
+import { Company } from './entity/Company';
 
 export const cio = new TrackClient(
   process.env.CIO_SITE_ID,
   process.env.CIO_API_KEY,
 );
 
+/**
+ * Specific client for using v2 of the Customer.io API.
+ */
+export const cioV2 = new TrackClient(
+  process.env.CIO_SITE_ID,
+  process.env.CIO_API_KEY,
+  { url: 'https://track.customer.io/api/v2' },
+);
+
+const CIO_COMPANY_OBJECT_ID = '4';
```

Comment:
What is 4? Magic number?

Reply:
yes, it's their internal identifier for this object.
That's why I abstracted it to a const declaration for naming it 😂

Reply:
```suggestion
// Magic number from customer.io
const CIO_COMPANY_OBJECT_ID = '4';
```

Reply:
Just a suggestion to add a comment saying it's a magic number from them.